docs/en/connector-v2/sink/Slack.md

```diff
@@ -0,0 +1,57 @@
+# Slack
+
+> Slack sink connector
+
+## Description
+
+Used to send data to Slack Channel. Both support streaming and batch mode.
+> For example, if the data from upstream is [`age: 12, name: huan`], the content send to socket server is the following: `{"name":"huan","age":17}`
+
+
+## Key features
+
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [schema projection](../../concept/connector-v2-features.md)
+
+## Options
+
+| name           | type   | required | default value |
+| -------------- |--------|----------|---------------|
+| webhooks_url   | String | Yes      | -             |
+| oauth_token    | String | Yes      | -             |
+| slack_channel  | String | Yes      | -             |
+| common-options |        | no       | -             |
+
+### webhooks_url [string]
+
+Slack webhook url
+
+### oauth_token [string]
+
+Slack oauth token used for the actual authentication
+
+### slack_channel [string]
+
+slack channel for data write
+
+### common options
+
+Sink plugin common parameters, please refer to [Sink Common Options](common-options.md) for details
+
+## Example
+
+```hocon
+sink {
+ SlackSink {
+  webhooks_url = "https://hooks.slack.com/services/xxxxxxxxxxxx/xxxxxxxxxxxx/xxxxxxxxxxxxxxxx"
+  oauth_token = "xoxp-xxxxxxxxxx-xxxxxxxx-xxxxxxxxx-xxxxxxxxxxx"
+  slack_channel = "channel name"
+ }
+}
+```
+
+## Changelog
+
+### 2.3.0-beta 2022-10-20
```

Comment:
`### 2.3.0-beta 2022-10-20` need replace with `new version`, `new version` will be update to the real version number when we release a version.

Reply:
Could you please tell me how long the new version will be released?Thx

Reply:
I have done it!